test/langtools/tools/javac/processing/model/type/AnnotatedTypeToString/AnnotatedTypeToString.java

```diff
@@ -0,0 +1,61 @@
+/*
```

Comment:
From personal experience with similar tests, they can look obscure when revisting them months in the future. I suggest putting a sentence or two description of what the test is trying to do. Also, if all the types can fit in one file, that can be easier to understand.